website/client/mixins/spells.js

```diff
@@ -32,9 +32,10 @@ export default {
           return this.castEnd(party, spell.target);
         }
 
-        let party = this.$store.state.partyMembers;
-        party = isArray(party) ? party : [];
-        party = party.concat(this.user);
+        let party = this.$store.state.partyMembers.data;
```

Comment:
can we rename this variable to partyMembers, it's quite confusing

Reply:
Sure